parsible.py

```diff
@@ -187,9 +204,13 @@ def main(self):
 
         # Abstract all the messy generator logic away into a simple for-each
         for parsed_line in parsed_log_file:
+            if(self.should_filter(parsed_line)):
```

Comment:
Nope, just that I'm new to python and a bit confused on some of it's syntax
:)

On Wed, Jun 10, 2015 at 6:46 AM, Andrew Gross <notifications@github.com>
wrote:

> In parsible.py
> <https://github.com/Yipit/parsible/pull/9#discussion_r32061831>:
>
> > @@ -187,9 +204,13 @@ def main(self):
> >
> >          # Abstract all the messy generator logic away into a simple for-each
> >          for parsed_line in parsed_log_file:
> > +            if(self.should_filter(parsed_line)):
>
> any particular reason for the () around self.should_filter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Yipit/parsible/pull/9/files#r32061831>.
>